core/src/main/java/info/openrocket/core/file/openrocket/OpenRocketSaver.java

```diff
@@ -374,7 +374,8 @@ private void saveSimulation(Simulation simulation, boolean saveSimulationData) t
 		writeElement("launchlatitude", cond.getLaunchLatitude());
 		writeElement("launchlongitude", cond.getLaunchLongitude());
 		writeElement("geodeticmethod", cond.getGeodeticComputation().name().toLowerCase(Locale.ENGLISH));
-		
+		writeElement("rungekuttamethod", cond.getRKStepperChoice().name().toLowerCase(Locale.ENGLISH));
```

Comment:
It would be better to make this more generic, e.g; "simulationsteppermethod". We may support other iterative methods in the future.

Reply:
You're fast, thanks 🙂